application/client.core/src/app/environment/components/sidebar/shell/information/template.html

```diff
@@ -0,0 +1,9 @@
+<mat-expansion-panel [expanded]="false">
+    <mat-expansion-panel-header>
+        <mat-panel-title>Information</mat-panel-title>
+    </mat-expansion-panel-header>
+    <div class="info">
+        <p>pwd: {{_ng_information.pwd}}</p>
+        <p>Shell: {{_ng_information.shell}}</p>
+    </div>
+</mat-expansion-panel>
```

Comment:
See a little above:
- spinner as a suggestion to reflect the progress of the IPC message
- error message in case of some unexpected error